applications_tests/lethe-fluid-vans-matrix-free/case1_vans_A.prm

```diff
@@ -0,0 +1,147 @@
+  set velocity order = 1
+  set pressure order = 1
+end
+
+#---------------------------------------------------
+# Void Fraction
+#---------------------------------------------------
+
+subsection void fraction
+  set mode = function
+  subsection function
+    set Function expression = 0.5 + 0.25*sin(pi*x)*sin(pi*y)
+  end
+end
+
+#---------------------------------------------------
+# CFD-DEM
+#---------------------------------------------------
+
+subsection cfd-dem
+  set grad div       = false
+  set vans model     = modelA
+  set pressure force = false
+  set shear force    = false
+end
+
+#---------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+
+subsection boundary conditions
+  set number = 1
+    set fix pressure constant = true
+
+  subsection bc 0
+    set id   = 0
+    set type = noslip
+  end
+end
+
+#---------------------------------------------------
+# Source term
+#---------------------------------------------------
+
+subsection source term
+  subsection fluid dynamics
+    set Function expression = pi*(-16*pi*sin(pi*x)^2*cos(pi*y) + 4*pi*cos(pi*y) + 4*sin(pi*x)^3*sin(pi*y)*cos(pi*x) + cos(pi*x))*sin(pi*y)  ;  pi*(16*pi*sin(pi*y)^2*cos(pi*x) - 4*pi*cos(pi*x) + 4*sin(pi*x)*sin(pi*y)^3*cos(pi*y) + cos(pi*y))*sin(pi*x)  ;  0
+  end
+end
+
+#---------------------------------------------------
+# Analytical Solution
+#---------------------------------------------------
+
+subsection analytical solution
+  set enable = true
+  set verbosity = verbose
+  subsection uvwp
+    set Function expression = -2*sin(pi*x) * sin(pi*x) * cos(pi*y) * sin(pi*y) ; 2*cos(pi*x) * sin(pi*x) * sin(pi*y) * sin(pi*y); sin(pi*x)*sin(pi*y)
+  end
+end
+
+#---------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+
+subsection mesh adaptation
+  set type = uniform
+end
+
+#---------------------------------------------------
+# Non-Linear Solver Control
+#---------------------------------------------------
+
+subsection non-linear solver
+  subsection fluid dynamics
+    set tolerance          = 1e-7
+    set max iterations     = 10
+    set residual precision = 2
+    set verbosity          = quiet
+  end
+end
+
+#---------------------------------------------------
+# Linear Solver Control
+#---------------------------------------------------
+
+subsection linear solver
+  subsection fluid dynamics
+  set method            = gmres
+  set max iters         = 1000
+  set relative residual = 1e-3
+  set minimum residual  = 1e-8
+  set preconditioner    = gcmg
+  set verbosity         = quiet
+  set max krylov vectors = 1000
+  set mg verbosity                   = quiet
+  set mg smoother iterations          = 1
+  set mg smoother eig estimation      = true
+  set mg smoother preconditioner type = additive schwarz method
+  set eig estimation smoothing range = 5
+  set eig estimation cg n iterations = 20
```

Comment:
Why do you set these two parameters for some tests and for others you use the default values? Is there any particular reason?

Reply:
No reason, I will put them all to default value I think